# A cylinder with its pose, radius, and extent.

geometry_msgs/Pose pose

float32 radius

float32 extent

# curvature axis (direction vector)
geometry_msgs/Vector3 axis
 
# normal axis (direction vector)
geometry_msgs/Vector3 normal
